ksql-cli/src/main/java/io/confluent/ksql/cli/console/cmd/SetCliProperty.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.cli.console.cmd;
+
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+final class SetCliProperty implements CliSpecificCommand {
+
+  private static final String HELP = "set cli <property> <value>:" + System.lineSeparator()
+      + "\tSets a CLI local property. NOTE that this differs from setting a KSQL "
+      + "property with 'SET property=value' in that it does not affect the server.";
```

Comment:
I think the existing SET requires quoted property name

```suggestion
      + "property with "SET 'property'=value" in that it does not affect the server.";
```